docs/reference/associated-topology.md

```diff
@@ -97,7 +97,7 @@ id: Helm:docs:example
 type: Helm
 ```
 
-The driver uses the name of the Helm release and the namespace it was installed in as a unique id and name (even though we are using Helm2, this prepares us for Helm3 where releases can have the same name but in different namespaces). 
+The driver uses the name of the Helm release and the namespace it was installed in as a unique id and name (this prepares us for Helm3 where releases can have the same name but in different namespaces). 
```

Comment:
The whole of the text about preparing for helm 3 can be removed here. The preparation is done.

Reply:
Hi @hhellyer , I have removed the text as per your comments